packages/legacy/core/App/components/misc/CredentialCard.tsx

```diff
@@ -46,41 +47,75 @@ const CredentialCard: React.FC<CredentialCardProps> = ({
   const { ColorPallet } = useTheme()
   const getCredOverlayType = (type: BrandingOverlayType) => {
     if (proof) {
-      return (
-        <CredentialCard11
-          displayItems={displayItems}
-          style={{ backgroundColor: ColorPallet.brand.secondaryBackground }}
-          error={!existsInWallet}
-          predicateError={!satisfiedPredicates}
-          credName={credName}
-          credDefId={credDefId}
-          schemaId={schemaId}
-          credential={credential as CredentialExchangeRecord}
-          handleAltCredChange={handleAltCredChange}
-          hasAltCredentials={hasAltCredentials}
-          proof
-          elevated
-        />
-      )
+      if (type === BrandingOverlayType.Branding11) {
+        return (
+          <CredentialCard12
+            displayItems={displayItems}
+            error={!existsInWallet}
+            style={{ backgroundColor: ColorPallet.brand.secondaryBackground }}
+            predicateError={!satisfiedPredicates}
+            credName={credName}
+            credDefId={credDefId}
+            schemaId={schemaId}
+            credential={credential as CredentialExchangeRecord}
+            handleAltCredChange={handleAltCredChange}
+            hasAltCredentials={hasAltCredentials}
+            proof
+            elevated
+          />
+        )
+      } else {
+        return (
+          <CredentialCard11
+            displayItems={displayItems}
+            style={{ backgroundColor: ColorPallet.brand.secondaryBackground }}
+            error={!existsInWallet}
+            predicateError={!satisfiedPredicates}
+            credName={credName}
+            credDefId={credDefId}
+            schemaId={schemaId}
+            credential={credential as CredentialExchangeRecord}
+            handleAltCredChange={handleAltCredChange}
+            hasAltCredentials={hasAltCredentials}
+            proof
+            elevated
+          />
+        )
+      }
     }
 
     if (credential) {
       if (type === BrandingOverlayType.Branding01) {
         return <CredentialCard10 credential={credential as CredentialExchangeRecord} style={style} onPress={onPress} />
+      } else if (type == BrandingOverlayType.Branding11) {
```

Comment:
I know this PR is still in draft but will it eventually have a `BrandingOverlayType.Branding12` and use that here, rather than use the 12 card with the 11 overlay type?

Reply:
I could do some refactor here @bryce-mcmath but in the oca package there is the Branding01, Branding10 and Branding11. The bifold component that implements each one of these are CredentialCard10 (For the legacy branding), CredentialCard11 (Branding10) and now CredentialCard12 (Branding11). 

I'm not sure why the components were not named respectively CredentialCard01, CredentialCard10 in the beginning (maybe due to lint reasons). Maybe we could name them:

Legacy -> CredentialCard01
Branding 1.0 -> CredentialCard10
Branding 1.1 -> CredentialCard11

Let me know what you think. Thanks!